app/components/createContainer/createContainerController.js

```diff
@@ -233,7 +234,157 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     return config;
   }
 
+  function confirmCreateContainer(cb) {
+    Container.query({
+      all: 1,
+      filters: {name: [$scope.config.name]}
+    }, function (data) {
+      var confirmDialog = false;
+      var container;
+      // Prompt if we found name to confirm replacement
+      for (var c in data) {
+        for (var n in data[c].Names) {
+          if (data[c].Names[n] === '/' + $scope.config.name) {
+            confirmDialog = true;
+            container = data[c];
+          }
+        }
+      }
+      if (confirmDialog) {
+        ModalService.confirmDeletion(
+          'A container with the same name is already present on this host. Do you want to remove it?',
+          function onConfirm(confirmed) {
+            if(!confirmed) { return cb(false); }
+            else {
+              // Remove old container
+              ContainerService.remove(container, true)
+              .then(function success(data) {
+                Notifications.success('Container Removed', container.Id);
+                return cb(true);
+              })
+              .catch(function error(err) {
+                Notifications.error('Failure', err, 'Unable to remove container');
+                return cb(false);
+              });
+            }
+          }
+        );
+      } else {
+        return cb(true);
+      }
+    }, function error(err) {
+      Notifications.error('Failure', err, 'Unable to retrieve containers');
+      return cb(false);
+    });
+  }
+
+  function loadFromContainerSpec() {
+    // Get container
+    Container.get({id: $stateParams.from}, function(d) {
+      $scope.fromContainer = new ContainerViewModel(d);
+      // Add Config
+      $scope.config = d.Config;
+      if ($scope.config.Cmd) {
+        $scope.config.Cmd = ContainerHelper.commandArrayToString($scope.config.Cmd);
+      }
+      // Add HostConfig
+      $scope.config.HostConfig = d.HostConfig;
+      // Add Ports
+      var bindings = [];
+      for (var p in $scope.config.HostConfig.PortBindings) {
+        var b = {
+          'hostPort': $scope.config.HostConfig.PortBindings[p][0].HostPort,
+          'containerPort': p.split('/')[0],
+          'protocol': p.split('/')[1]
+        };
+        bindings.push(b);
+      }
+      $scope.config.HostConfig.PortBindings = bindings;
+      // Add volumes
+      for (var v in d.Mounts) {
+        var mount = d.Mounts[v];
+        var volume = {
+          'type': mount.Type,
+          'name': mount.Name || mount.Source,
+          'containerPath': mount.Destination,
+          'readOnly': mount.RW === false
+        };
+        $scope.formValues.Volumes.push(volume);
+      }
+      // Add network
+      $scope.config.NetworkingConfig = {
+        EndpointsConfig: {}
+      };
+      if ($scope.config.HostConfig.NetworkMode.indexOf('container:') === 0) {
+        $scope.formValues.NetworkContainer = $scope.config.HostConfig.NetworkMode.split(/^container:/)[1];
```

Comment:
Well, this is going to be tricky as the value `$scope.formValues.NetworkContainer` should be a container object (as returned by the Docker API) and not a simple string (container name).

You can iterate through the containers available in this view via `$scope.runningContainers` to retrieve the container object based on it's name.